internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -207,6 +207,12 @@ func (r *ISBServiceRolloutReconciler) reconcile(ctx context.Context, isbServiceR
 			if err := r.client.Delete(ctx, isbServiceRollout, &client.DeleteOptions{PropagationPolicy: &foreground}); err != nil {
 				return ctrl.Result{}, err
 			}
+			// Get the nfcRollout live resource
+			LiveISBServiceRollout, err := kubernetes.NumaplaneClient.NumaplaneV1alpha1().ISBServiceRollouts(isbServiceRollout.Namespace).Get(ctx, isbServiceRollout.Name, metav1.GetOptions{})
```

Comment:
any reason for the capital letter "L"? or just a typo?

Reply:
Oh! it's just a typo